paasta_tools/check_oom_events.py

```diff
@@ -104,49 +115,38 @@ def latest_oom_events(cluster, superregion, interval=60):
     for e in read_oom_events_from_scribe(cluster, superregion):
         if e["timestamp"] > start_timestamp:
             key = (e["service"], e["instance"])
-            res.setdefault(key, []).append(
-                OOMEvent(
-                    hostname=e.get("hostname", ""),
-                    container_id=e.get("container_id", ""),
-                    process_name=e.get("process_name", ""),
-                )
-            )
+            res.setdefault(key, set()).add(e.get("container_id", ""))
     return res
 
 
-def compose_sensu_status(instance, oom_events, is_check_enabled):
+def compose_sensu_status(
+    instance, oom_events, is_check_enabled, alert_threshold, check_interval
+):
     """
     :param instance: InstanceConfig
     :param oom_events: a list of OOMEvents
     :param is_check_enabled: boolean to indicate whether the check enabled for the instance
     """
+    interval_string = f"{check_interval} minute(s)"
+    instance_name = f"{instance.service}.{instance.instance}"
     if not is_check_enabled:
-        return (
-            Status.OK,
-            "This check is disabled for {}.{}.".format(
-                instance.service, instance.instance
-            ),
-        )
+        return (Status.OK, f"This check is disabled for {instance_name}.")
     if len(oom_events) == 0:
```

Comment:
If we send None instead of an empty list as oom_event into the function for some reason len(None) going to err.